File: app/src/main/java/omniex/nl/omniex/ui/app/categories/CategoriesView.java
package omniex.nl.omniex.ui.app.categories;

import java.util.List;

import nl.omniex.omniexshopping.data.model.products.Category;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface CategoriesView extends BaseView {
    void onCategoriesFetched(List<Category> categoryList);
}

File: app/src/main/java/omniex/nl/omniex/ui/app/categories/CategoriesView.kt
package omniex.nl.omniex.ui.app.categories


import omniex.nl.omniex.data.model.products.Category
import omniex.nl.omniex.ui.base.BaseView

interface CategoriesView : BaseView {
    fun onCategoriesFetched(categoryList: List<Category>)
}
